app/views/record/_extended_info.html.erb

```diff
@@ -49,21 +49,19 @@
     </li>
   <% end %>
 
-  <% if @record.eds_isbn_print.present? %>
+  <% if @record.eds_isbns.present? %>
     <li>
-      ISBN: <%= @record.eds_isbn_print %>
+      ISBN: <% @record.eds_isbns.each do |isbn| %>
+        <%= isbn %> <br />
+      <% end %>
     </li>
   <% end %>
 
-  <% if @record.eds_isbn_electronic.present? %>
+  <% if @record.eds_issns.present? %>
     <li>
-      ISBN: <%= @record.eds_isbn_electronic %>
-    </li>
-  <% end %>
-
-  <% if @record.eds_issn_print.present? %>
-    <li>
-      ISSN: <%= @record.eds_issn_print %>
+      ISSN: <% @record.eds_issns.each do |issn| %>
+        <%= issn %> <br />
```

Comment:
Can you remove this hard line break and wrap the isbn in `<span class="isbn"></span>`